src/test/java/uk/gov/pay/adminusers/utils/email/EmailValidatorIsPublicSectorEmailTest.java

```diff
@@ -129,13 +129,14 @@
                 {"valid@subdomain.wmca.org.uk", true},
                 {"valid@subdomain.york.ac.uk", true},
                 {"valid@subdomain.digitalaccessibilitycentre.org", true},
+                {"valid@subdomain.fermanaghomagh.com", true},
```

Comment:
There should be too of these: `valid@fermanaghomagh.com` (in this group higher up) and `valid@subdomain.fermanaghomagh.com` (and, again, in alphabetical order).